lib/sidebar_builder.rb

```diff
@@ -0,0 +1,127 @@
+# frozen_string_literal: false
+
+module SwedbankPay
+  # A nice sidebar
+  class SidebarBuilder
+    def initialize(filename, pages)
+      @filename = filename
+      @pages = pages
+    end
+
+    def build
+      sidebar_markup = ''
+
+      @pages.select { |path, _| path.split('/').length <= 2 }.each do |path, page|
```

Comment:
Codacy found an issue: [Line is too long. [82/80]](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=6198407)